router/routes/pew.js

```diff
@@ -14,6 +14,9 @@ router.post('/', function(request, response) {
   var input = request.body.text;
   var token = request.body.token;
   var message = ["<!group>: PEW"];
+  var parse = [];
```

Comment:
Usually you use the imperative to name methods and past tense for results. In this case, it should be `parsed`.

```js
var parsed = parse(blah);
```